internal/testutil/keys.go

```diff
@@ -21,21 +21,21 @@ import (
 	"github.com/mailchain/mailchain/crypto/secp256k1"
 )
 
-var SofiaPrivateKey crypto.PrivateKey     // nolint: gochecknoglobals
-var SofiaPublicKey crypto.PublicKey       // nolint: gochecknoglobals
-var CharlottePrivateKey crypto.PrivateKey // nolint: gochecknoglobals
-var CharlottePublicKey crypto.PublicKey   // nolint: gochecknoglobals
+var SofiaPrivateKey crypto.PrivateKey     //nolint: gochecknoglobals
+var SofiaPublicKey crypto.PublicKey       //nolint: gochecknoglobals
+var CharlottePrivateKey crypto.PrivateKey //nolint: gochecknoglobals
+var CharlottePublicKey crypto.PublicKey   //nolint: gochecknoglobals
 
-// nolint: gochecknoinits
+//nolint: gochecknoinits
 func init() {
 	var err error
-	SofiaPrivateKey, err = secp256k1.PrivateKeyFromHex("01901E63389EF02EAA7C5782E08B40D98FAEF835F28BD144EECF5614A415943F")
+	SofiaPrivateKey, err = secp256k1.PrivateKeyFromBytes("01901E63389EF02EAA7C5782E08B40D98FAEF835F28BD144EECF5614A415943F")
```

Comment:
`secp256k1.PrivateKeyFromBytes` is expecting `[]byte` You need to first do a `hex.DecodeString("01901E63389EF02EAA7C5782E08B40D98FAEF835F28BD144EECF5614A415943F")` check the error then pass it to  `secp256k1.PrivateKeyFromBytes`.